### PR TITLE
[Telink] Add B92 (tlsr9528a) SoC support & Update Telink image to 10

### PR DESCRIPTION
**Pull request by s07641069**
**Change overview**
- Update Telink image to 10 https://github.com/project-chip/connectedhomeip/pull/28801
- Add B92 (tlsr9528a) SoC support
- Add B92 (tlsr9528a) to CI
- Remove set board by default
- Set correct BOOT_MAX_IMG_SECTORS for B92
- Update partitions
- Reset default CONFIG_SETTINGS_NVS_SECTOR_SIZE_MULT
- Add support of Telink USB dongle
- Add Chef example to CI

### CODE REVIEW

File: .github/workflows/examples-telink.yaml [hunk 1]
```diff
@@ -278,3 +278,255 @@ jobs:
               if: ${{ !env.ACT }}
               with:
                 platform-name: Telink
+
+    tlsr9528a:
```
[the remark this answers is not in the document]
Comment (s07641069) [reply .github/workflows/examples-telink.yaml:282]: B91 and B92 build using separate CI sequences (runs in parallel).
We can exclude build of some examples for B92, but anyway B91 will take nearly 2 hours and general Telink CI duration will be the same.

[the remark this answers is not in the document]
Comment (s07641069) [reply .github/workflows/examples-telink.yaml:282]: looks like to speed-up Telink CI we have to divide B91 pipeline into 2 parallel as well as B92 after (will have 4 parallel parts).
What do you think?

[the remark this answers is not in the document]
Comment (s07641069) [reply .github/workflows/examples-telink.yaml:282]: @andy31415 Could you explain in more details what exactly I have to change/improve?

[the remark this answers is not in the document]
Comment (andy31415) [reply .github/workflows/examples-telink.yaml:282]: Github limits the number of runners from a central pool, so even if total runtime is in parallel, this is adding 2 hours of run time for all PRs from a shared pool.

[the remark this answers is not in the document]
Comment (s07641069) [reply .github/workflows/examples-telink.yaml]: Divideв CI apps between two SoCs

**Review by andy31415 — APPROVED** [review]

**Pull request merged**